src/sentry/event_manager.py

```diff
@@ -1154,12 +1159,13 @@ def _get_or_create_group_release(
 ) -> None:
     if release:
         for group_info in groups:
-            group_info.group_release = GroupRelease.get_or_create(
-                group=group_info.group,
-                release=release,
-                environment=environment,
-                datetime=event.datetime,
-            )
+            if not can_create_group(group_info.group, group_info.group.project):
```

Comment:
I think this check is causing problems - We actually use `_get_or_create_group_release` in the issues ingest pipeline as well, and separately it's causing problems with errors. I think it's fine to rely on not adding the groups in `_save_aggregate_performance` instead, going to try removing

Reply:
oh that's what it was. yeah I saw all these failures